### DISCUSSION TITLE
在Pycharm的终端中出现乱码

### DISCUSSION THREAD

**Discussion by doiya-xx** [nonebot/discussions#327] [问答] [answered]
## 系统配置

- windows 11
- Pycharm 2023.3
- python 3.10
- nb-cli

## 问题

在Pycharm中打开项目，启动Pycharm中的终端。输入`nb`命令。当开始移动鼠标或者触控板时，就会出现移动代码。

```
# (py310) PS E:\Codespace\PythonProjects\nonebot-lxmon> nb

欢迎使用 NoneBot CLI!
[?] 你想要进行什么操作? (Use ↑ and ↓ to choose, Enter to submit) #!@"!@!!@! #! 
```

在`terminal`或者`VSCode`的终端中运行`nb`命令则不会出现这种情况。

**Comment by yanyongyu** [comment, marked as answer]
这问题你得问pycharm
